extensions/country-lookup/package.json

```
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "country-lookup",
+  "title": "Country Lookup",
+  "description": "Quickly lookup countries and get informational data",
+  "icon": "command-icon.png",
+  "author": "pernielsentikaer",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Country Lookup",
```

Not sure about subtitle here, but I think it would be better:
```suggestion
      "title": "Search Country",
```